README.md

```diff
@@ -26,3 +30,5 @@ In order to re-run an analysis stored here, follow the following steps:
 # License anc citation
 
 This repository is a compilation of different projects, each one with its own license. The same applies to its citation. Please check the license and citation of each project before using it.
+
+If no licence is specified, the MIT licence of this repository applies.
```

Comment:
This it isn't really true, you cannot "transfer" licenses from linked reposutories. Could we rephrase the sentence to something like
"This repository is licensed under the MIT license. Please be aware that the license of this repository does not apply to the linked repositories submodules."